reinforcement_learning/rl_deepracer_robomaker_coach_gazebo/deepracer_rl.ipynb

```diff
@@ -1127,8 +1127,9 @@
     "s3_bucket_2 = sage_session.default_bucket()\n",
     "\n",
     "# Ensure that the S3 prefix contains the keyword 'sagemaker'\n",
-    "# s3_prefix_2 = \"deepracer-notebook-sagemaker-200422-231836\"\n",
-    "s3_prefix_2 = \"deepracer-notebook-sagemaker-200422-231836\"\n",
+    "# e.g.\"deepracer-notebook-sagemaker-200422-231836\"\n",
+    "# Please provide the second agents s3_prefix\n",
+    "s3_prefix_2 = \"[FILL OUT PREFIX]\"\n",
```

Comment:
where is FILL OUT PREFIX defined?

Reply:
@yijiezh 
Can you help here?

Reply:
This is dummy cell. FILL OUT PREFIX is not defined in the notebook. Customer need to replace [FILL OUT PREFIX] with their own bucket prefix